lib/server/hapi_plugins/api/internals.js

```diff
@@ -11,6 +11,8 @@ module.exports = {
       return;
     }
 
+    var isSess = request.method === 'post' && request.path === '/_api/_session';
```

Comment:
The condition described in this [comment](https://github.com/hoodiehq/hoodie-server/pull/355#issuecomment-119677979).